### ISSUE TITLE
Functions that need more definition

### ISSUE THREAD

**Issue by aaron0browne** [PEDSnet/pedsnetdcc#13]
- "execute things in parallel"
- "iterate over multiple sites for a given command" (parallel as an option)
- "handle NOT NULL constraints"
- "move primary key from old table name to new table name"
- "Undo transformations" - create table as select stock tables

**Comment by aaron0browne** [comment]
For items 3 and 4 see this old script: https://github.com/PEDSnet/dcc-loader/blob/master/id-map/replace-tables.sql

**Comment by aaron0browne** [comment]
Item 1 is done.

**Comment by aaron0browne** [comment]
Item 3 is described in #27.

**Comment by aaron0browne** [comment]
Item 2 is described in #28.

**Comment by aaron0browne** [comment]
Item 4 is described in #29.

**Comment by aaron0browne** [comment]
Item 5 is described in #31.

**Comment by aaron0browne** [comment]
Closing in favor of mentioned issues.
